plugins/woocommerce/tests/e2e/config/jest.setup.js

```diff
@@ -40,6 +40,8 @@ async function trashExistingPosts() {
 // each other's side-effects.
 beforeAll(async () => {
 
+	setupJestObject();
```

Comment:
We could enable retries by default to changing this to `setupJestRetries( X )` where X is 2 or 3 to produce a total of 3 tries.

For the e2e-environment package `jest.setup.js` we can add `setupJestRetries()` to the `beforeAll()`.

Reply:
Great suggestions! I've made the necessary changes.